### PR TITLE
📖 Update Placement, Locations, and Scheduling doc

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR attempts to make some udpates to bring that doc more in line with current truth.  It only goes as far as I understand right now.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
@pdettori

File: docs/content/concepts/placement-locations-and-scheduling.md [hunk 1]
```diff
@@ -170,12 +173,14 @@ workspace.
...
-kcp would need to receive a resource that was provisioned by a controller on the `SyncTarget`.
-This is the case with storage PVs, which are created on the `SyncTarget` by a CSI driver.
+kcp would need to receive a resource that was provisioned by a controller on the pcluster.
+This is the case with storage PVs, which are created on the pcluster by a CSI driver.
 
 Unlike the `Sync` state, the `Upsync` state is exclusive, and only a single `SyncTarget` can be the source of truth for an upsynced resource.
 In addition, other `SyncTargets` cannot be syncing down while the resource is being upsynced.
 
+Upsyncing is initiated by something creating a label with key = `state.workload.kcp.io/<sync-target-key>` and value = `UpSync` on the object in the pcluster.
+
```
Comment (pdettori) [inline docs/content/concepts/placement-locations-and-scheduling.md:183]: The value is "Upsync" (with lowercase "s") as can be seen in https://github.com/kcp-dev/kcp/blob/dc49b99881374f748949250a8ff3b737f019ef0c/pkg/apis/workload/v1alpha1/types.go#L34

**Review by pdettori — CHANGES_REQUESTED** [review]

File: docs/content/concepts/placement-locations-and-scheduling.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/concepts/placement-locations-and-scheduling.md:183]: Thanks. Fixed in the force-push to d5b1b8c3

**Review by ncdc — APPROVED** [review]

**Pull request merged**